docs/static/security/es-security.asciidoc

```diff
@@ -2,11 +2,15 @@
 [[es-security-on]]
 == {es} security on by default
 
-{es} {ref}/configuring-stack-security.html[security is on by default] starting in 8.0.
+Security is on by default starting in 8.0.
 {es} generates its own default self-signed Secure Sockets Layer (SSL) certificates at startup. 
```

Comment:
This line is redundant with line 10, and provides an unguarded blanket statement that will not be true for many of our security-conscious users who are providing their own PKI.